### PR TITLE
README.md: Remove Spacing Inconsistency

### PR DESCRIPTION
**Pull request by sladyn98**
Readme.md contained some spacing inconsitency
which has been now removed.

Fixes https://github.com/coala/projects/issues/713

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,12 +44,6 @@ before_install:
 # the `install` step will run `bundle install` by default.
 script:
   - .ci/build.sh
-  # https://github.com/coala/coala-bears/issues/1037
```
Comment (frextrite) [inline .travis.yml]: Why has this section been removed?

File: .travis.yml [hunk 2]
```diff
@@ -22,6 +22,19 @@ jobs:
       cache: pip
       install: pip install moban
       script: .ci/check_moban.sh
+    - stage: coala
```
Comment (frextrite) [inline .travis.yml]: and this added?

These changes are not related to the `SpaceConsistencyBear` issue afaik

Comment (sladyn98) [reply .travis.yml]: No your right these sections are not related. This is related to another issue https://github.com/coala/projects/pull/600. Actually makman2 and me were planning to fix the CI in this pr itself because gsoc is already upon us. I have discussed this with him and based on that appropriate action will be taken :)

Comment (frextrite) [reply .travis.yml]: That's great. Just add that issue reference to the comment/pr as well.

File: .travis.yml [hunk 3]
```diff
@@ -22,6 +22,19 @@ jobs:
...
+      <<: *disable_global
+      language: python
+      python: 3.6
+      cache: pip
+      install:
+        - cOALA_ZIP_URL=https://github.com/coala/coala/archive/master.zip
```
Comment (frextrite) [inline .travis.yml]:
```suggestion
        - COALA_ZIP_URL=https://codeload.github.com/coala/coala/archive/master.zip
```

**Comment by abhishalya** [comment]
@sladyn98 I would've really liked if there is a single PR fixing the CI. Two PRs for same purpose makes one redundant. Although this PR references changes regarding space consistency but the goal is same. Anyways, as soon as https://github.com/coala/projects/pull/600 is finished, you need to cherry pick that commit and push here. We should be able to see the CI green here, since that commit is has been done before than this (following linear git history).

File: _reports/GuangyuanYang.md [hunk 4]
```diff
@@ -164,7 +164,7 @@ investigation/review. Based on this feature, I implemented a
 [script](https://gist.github.com/yzgyyang/13ebb46a5c90ac624b5c6eb43deedc0d) to
 demonstrate the ability of bootstrapping coala Go bears on a bare-metal FreeBSD
 system
-([Screen record for the demo](https://asciinema.org/a/0poVbHE7lqehV4islFo6cptjb)),
+([Screen record for the demo]()),
```
Comment (abhishalya) [inline _reports/GuangyuanYang.md]: I don't think so this is required. This was reporting InvalidLink because we were using old coala. You should run latest coala and verify if it still throws InvalidLink.

**Comment by sladyn98** [comment]
> @sladyn98 I would've really liked if there is a single PR fixing the CI. Two PRs for same purpose makes one redundant. Although this PR references changes regarding space consistency but the goal is same. Anyways, as soon as #600 is finished, you need to cherry pick that commit and push here. We should be able to see the CI green here, since that commit is has been done before than this (following linear git history).

Yeah @abhishalya your bang on I will remove the changes from my PR once #600 has been fixed :)

File: _projects/use-coala.md [hunk 5]
```diff
@@ -64,14 +64,14 @@ so that coala can check most file types.
 
 Find a GitHub repository that meets all these criteria:
 
-1. 100 or more [stars](https://help.github.com/articles/about-stars/),
+1. 100 or more [stars](https://help.github.com/articles/about-stars),
```
Comment (abhishalya) [inline _projects/use-coala.md]: Are these changes related to SpaceConsistency?

Comment (sladyn98) [reply _projects/use-coala.md]: No the link bear complained even after we installed the latest version

File: _reports/GuangyuanYang.md (hunk 4 above)
Comment (sladyn98) [reply _reports/GuangyuanYang.md]: I just followed up on @kriti21's PR and I think she installed the latest coala and still the bear complained.

**Comment by sladyn98** [comment]
@abhishalya @kriti21  https://travis-ci.org/sladyn98/projects/builds/511483343 I have linked the branch which made the CI green on my local fork you can have a look.The one problem is the asci link. I do not think I should have made that change because we just cannot remove a link from someone's project.So we need to find a workaround that.

File: _projects/use-coala.md (hunk 5 above)
Comment (abhishalya) [reply _projects/use-coala.md]: Then I think its better to add a new commit for this change.

File: .travis.yml (hunk 1 above)
Comment (jayvdb) [reply .travis.yml]: See  @kriti21's PR.

@sladyn98 	, i suggest you make your PR only about fixing the spacing.  Leave the other problem to kriti.

Or split your commit into two well written commits, as they are two distinct fixes.

Comment (sladyn98) [reply .travis.yml]: @jayvdb yep agreed I made these changes intially with the aim of having my PR merged but since @kriti21 got her CI green I am removing these changes :+1:

**Comment by sladyn98** [comment]
@jayvdb I guess this is done, and ready to be merged.?

**Comment by sladyn98** [comment]
@jayvdb This fixes the spacing inconsistency and is ready to be merged IMO.

**Comment by jayvdb** [comment]
Duplicate of #600
